src/service/poc_mobile.proto

```diff
@@ -308,6 +336,8 @@ enum cell_type {
   neutrino430 = 2;
   sercomm_indoor = 3;
   sercomm_outdoor = 4;
+  cell_type_none = 5;
+  nova_generic_wifi_indoor = 6;
```

Comment:
do we need `generic`? is missing that distinction a meaningful thing for the `secomm_indoor/outdoor` cbrs radios above?

Reply:
generic was agreed so that we can permit additional future models to be added